pydatalab/pydatalab/apps/nmr/blocks.py

```diff
@@ -17,7 +17,7 @@ class NMRBlock(DataBlock):
     name = "NMR"
     description = "A simple NMR block for visualizing 1D NMR data from Bruker projects."
 
-    accepted_file_extensions = ".zip"
+    accepted_file_extensions = (".zip",)
```

Comment:
Nice catch, this should have been flagged by mypy but we're using a weird type hint on the base class (`Sequence[str]`) -- I'll push a commit that fixes it to (`tuple[str, ...]`) this PR and see if anything else comes out